src/app/shared/services/employee.service.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import DataSource from 'devextreme/data/data_source';
+import { Observable } from 'rxjs';
+
+
+@Injectable({
+  providedIn: 'root',
+})
+export class EmployeeService {
+  
+  list =[ ]
+
+  constructor(private _http: HttpClient) {}
+
+  
+
+  addEmployee(data: any): Observable<any> {
+    return this._http.post('http://localhost:3000/employee', data);
```

Comment:
make this URL configurable, so tomorrow if host server changes, it should be easy to update